astro/src/content/docs/lifecycle/authenticate-users/identity-providers/gaming/xbox.mdx

```diff
@@ -28,11 +29,11 @@ Once you have completed this configuration you will be able to enable the Xbox l
 
 <img src="/img/docs/lifecycle/authenticate-users/identity-providers/gaming/login-xbox.png" alt="Xbox Login" role="bottom-cropped" width="1200" />
 
-## Create an Xbox Account
+<IdentityProviderOverviewDiagram identity_provider_name="XBox" />
```

Comment:
I'm being totally picky and should `XBox` be `Xbox`?

Reply:
Great catch. Looks like we use `Xbox` for our docs, so let me correct that.